eslint/index.js

```diff
@@ -4,13 +4,15 @@ module.exports = {
     "amd": true,
     "es6": true,
     "shared-node-browser": true,
-    "jest": true
+    "jest": true,
+    "browser": true
   },
   "parser": "babel-eslint",
   "plugins": [
     "import",
     "react",
-    "filenames"
+    "filenames",
+    "whatwg-fetch"
```

Comment:
@kylealwyn 
Talked with @berfarah . I'll remove the `whatwg-fetch` dependency and add will instead add a global rule for it in the main app.